### DISCUSSION TITLE
Using MICOM to model microbiome response to food challenge

### DISCUSSION THREAD

**Discussion by PathogeNish** [micom-dev/micom#150] [General]
I was wondering whether MICOM can be used to model microbial response to foods or diets. While I understand that this might not have been the intended use for MICOM (and I do see some challenges with the approach proposed below), I hope to get your opinions on how we can modify it for this use case.

### What MICOM does:
(_Please let me know if my understanding is incorrect anywhere_)

> MICOM estimates the concentrations of metabolite fluxes in the gut by modelling the gut microbial community with a set of input metabolite fluxes (representing food consumed as part of a diet). Additionally, MICOM performs a numerical optimisation to find microbial growth rates that maximise regularised community growth rate. Using these growth rates, MICOM then performs FBA (assuming the otu abundance table is at a dynamic steady state) on biomass equations for each microbial model to estimate the total concentration of metabolite fluxes produced/ consumed by the microbiome.

### Consider the following:

> Given an individual's diet (e.g. western diet), we want to know what happens to the microbes and metabolite fluxes when we increase the amount of, say, cheese in their diet. Let W be the standard western diet and W<sup>CH</sup> be the western diet with an additional 100gm of cheese included. Let G be the growth rates of microbes in a given microbiome abundance table M, under W and G<sup>CH</sup> be their growth rates under W<sup>CH</sup>. 

### Questions
1. How can we interpret changes in G and G<sup>CH</sup>?
2. How can we interpret corresponding metabolic profiles under the original and perturbed diet?

Is it possible to comment on the (instantaneous) direction of change of microbiome M under the additional influence of 100gm of cheese?

I would like to immediately highlight an issue with this line of thinking: MICOM only models fluxes at steady state. Modelling M with W<sup>CH</sup>, assumes, by definition, that M is stable under W. So there is an inherent issue in measuring the effect of perturbations in a model that is designed to operate under a steady state assumption.

I would like to hear any and all thoughts about this. Maybe its possible to conceive of a variation to MICOM or a completely new modelling technique that can allow us to model such microbiome responses to food challenges?

**Comment by cdiener** [comment]
Hi, your summary is correct. There are some details in the medium formulation because diet gives you (relative) concentrations and we have to convert those to fluxes with some approximations. We usually also add in some host compounds (bile acids, mucins) and deplete metabolites absorbed in the small intestine. The [MICOM media repository](https://github.com/micom-dev/media) has a bunch of examples.

What you are proposing is feasible and we have shown that this can in principle work (see for instance [here](https://www.biorxiv.org/content/10.1101/2023.02.28.530516v2)). To add cheese you would either have to use a tool like the [VMH diet designer](https://www.vmh.life/#nutrition/dietdesigner) or map the food to the metabolites in your model database by hand.

1. In terms of the growth rates you are right that there is a steady state assumption which would mean that for the abundance to remain constant dilution/death and growth rate would be balanced. If a food supplementation alter the growth rate and we would assume the dilution is remaining the same that would mean that the change in growth rate now leads to an abundance increase/decrease of the taxon. So you can use G_CH - G as en estimate of the predicted abundance change close to the steady state (similar to what dFBA does for example). One caveat is that by adding food you also increase the energy content in the medium so the base assumption would be that some taxa increase their growth rates because more nutrients are available.

2. Similar for fluxes. You could track the fluxes and see where the metabolites from your added food go and what they are converted into.
